src/components/ebay-checkbox/checkbox.stories.js

```diff
@@ -71,7 +71,7 @@ export default {
             table: {
                 category: "Events",
                 defaultValue: {
-                    summary: "{ originalEvent, value }",
+                    summary: "{ originalEvent, value, checked }",
```

Comment:
Not passing the component class as param here. Do we need to pass it? I don't see any value.

Reply:
That's due to storybook implementation